R/describe-feature.R

```diff
@@ -44,7 +44,7 @@ bcdc_describe_feature <- function(record = NULL){
   ## GET and parse data to sf object
   cli <- bcdc_http_client(url = "https://openmaps.gov.bc.ca/geo/pub/wfs")
 
-  cc <- cli$get(query = query_list)
+  cc <- cli$post(body = query_list, encode = "form")
```

Comment:
Do you think it's worth making a small utility function since this is used a bunch of times?

```r
post_it <- function(cli, body = NULL) cli$post(body = body, encode = "form")
```

Then:
```r
cc <- post_it(cli, query_list)
```

Reply:
Though see my other question below...

Reply:
I wonder if it is possible here to define a new R6 method within our package that works on a crul object.